src/config/options.rs

```diff
@@ -216,6 +216,97 @@ pub enum Verbosity {
     Quiet,
 }
 
+/// Which comments to wrap
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Serialize)]
+pub enum WrapComments {
+    /// Don't wrap comments
+    Off,
+    /// Wrap all kinds of comments
+    All,
+    /// Only wrap doc comments
+    Doc,
+    /// Only wrap normal comments
+    Normal,
+}
```

Comment:
In addition to moving the `ConfigType` impl, let's also move `WrapComments` and the other impls into a separate module. I'm thinking `src/config/wrap_comments.rs`